src/_includes/layouts/posts/singlepost.11ty.js

```
@@ -45,17 +45,17 @@ exports.render = function (data) {
   ${data.title != "The obligatory About Me page"
     ? `<div class="bg-dark">
     <h3 class="ctr wht"><a href="/posts" style="border-bottom: 0;">Other posts</a></h3>
-    ${nextPost.url !== null
+    ${data.nextPost && data.nextPost.url !== null
```

Needs to be `data.` prefixed, because `nextPost` isn't in scope (and is passed via the data object).
Also, when `data.nextPost` is undefined, the call to `.url` implodes because we're trying to access a property of an undefined object. So I added a quick check for `data.nextPost` so it can fail early.

---

@pdehaan Very much appreciated, sir.